org.flatpak.popout3d.yml

```diff
@@ -0,0 +1,196 @@
+app-id: com.github.PopoutApps.popout3d
+
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+
+command: popout3d
+rename-desktop-file: popout3d.desktop
+rename-icon: popout3d_logo
+copy-icon: true
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=host:rw
+modules:
+
+  - name: python3-pillow
+    buildsystem: simple
+    build-commands:
+      - pip3 install --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} Pillow Pillow-PIL
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/3c/7e/443be24431324bd34d22dd9d11cc845d995bcd3b500676bcf23142756975/Pillow-5.4.1.tar.gz
+        sha256: 5233664eadfa342c639b9b9977190d64ad7aca4edc51a966394d7e08e7f38a9f
+      - type: file
+        url: https://files.pythonhosted.org/packages/5d/e1/d298934cd78984a5e8d616945349ed66a74b9df0337fb1941ce76b8a2b7b/Pillow-PIL-0.1dev.tar.gz
+        sha256: d80ee30805f10bf898df9ce7495f46e88990e8f90b5c2b917722d87526252b2c
+
+  # Hugin deps  
+  - name: glu
```

Comment:
you should pull in the shared-modules versions of glew and glu https://github.com/flathub/shared-modules

Reply:
I've done that, but it fails, I don't know whether the failure is related to this change.
...
config.status: executing libtool commands
  CC       src/libutil/glue.lo
  CC       src/libutil/error.lo
  CC       src/libutil/mipmap.lo
src/libutil/error.c:60:10: error: ‘NULL’ undeclared here (not in a function)
   60 |    { ~0, NULL } /* end of list indicator */
      |          ^~~~
src/libutil/error.c:34:1: note: ‘NULL’ is defined in header ‘<stddef.h>’; did you forget to ‘#include <stddef.h>’?
   33 | #include <GL/glu.h>
  +++ |+#include <stddef.h>
   34 | 
make: *** [Makefile:1119: src/libutil/error.lo] Error 1
make: *** Waiting for unfinished jobs....
Error: module glu: Child process exited with code 2